extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -30,25 +30,31 @@ public function init() {
 	public function index() {
 		wfProfileIn( __METHOD__ );
 
+		if ( $this->areForumsArchivedAndDiscussionsEnabled() ) {
+			$this->redirectToDiscussions();
+			wfProfileOut( __METHOD__ );
+
+			return;
+		}
+
 		$output = $this->getContext()->getOutput();
 
 		$output->setRobotPolicy( "index,follow" );
 
 		$policies = Title::newFromText( 'forum-policies-and-faq', NS_MEDIAWIKI );
-		$this->response->setJsVar( 'wgCanEditPolicies', $this->wg->User->isAllowed( 'forumadmin' ) );
-		$this->response->setJsVar( 'wgPoliciesRev', $policies->getLatestRevID()  );
-		$this->response->setJsVar( 'wgPoliciesEditURL', $policies->getFullUrl( 'action=edit' )  );
+		$this->response->setJsVar( 'wgCanEditPolicies',
+			$this->wg->User->isAllowed( 'forumadmin' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "forumadmin" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)